### ISSUE TITLE
account banned

### ISSUE THREAD

**Issue by thearthouse** [seregazhuk/php-pinterest-bot#507]
hi i lost my account with this bot. i want to change some of my pins title.  but after   i received an email "Your Pinterest account has been suspended" .  serega you kill me!

**Comment by Iquaridys** [comment]
of course, you can lose your account because of this public tool, even Pinterest Crew can see this tool. You don't consider that?
